php_phongo.c

```diff
@@ -2583,6 +2583,16 @@ PHP_MINFO_FUNCTION(mongodb)
 #else
 	/* Bundled libraries, buildtime = runtime */
 	php_info_print_table_row(2, "libmongoc version", MONGOC_VERSION_S);
+# if HAVE_MONGOC_SSL
```

Comment:
So when compiling against system mongoc these well say no sasl and ssl support.
That doesn't look right.